src/vscodeAdapter.ts

```diff
@@ -999,4 +999,16 @@ export interface vscode {
         sessionId: string;
         openExternal(target: Uri): Thenable<boolean>;
     };
+
+    l10n: {
```

Comment:
We still need this type-
1.  The old mocha tests (and the production code being tested) still require it to run.
2.  The new jest tests currently require it to define the fake 'vscode' to export.  We can potentially replace this fake with real mocks once everything has migrated over.